google/cloud/bigquery/v2/minimal/internal/job_rest_stub.cc

```diff
@@ -49,17 +52,31 @@ StatusOr<GetJobResponse> DefaultBigQueryJobRestStub::GetJob(
     }
   }
 
-  // Call the rest client to get job details from the server as a RestResponse.
-  auto rest_response = rest_stub_->Get(std::move(*rest_request));
-  if (!rest_response) return std::move(rest_response).status();
+  return rest_request;
+}
 
-  // Convert RestResponse to HttpResponse.
-  auto http_response =
-      BigQueryHttpResponse::BuildFromRestResponse(std::move(*rest_response));
-  if (!http_response) return std::move(http_response).status();
+StatusOr<GetJobResponse> DefaultBigQueryJobRestStub::GetJob(
+    rest_internal::RestContext& rest_context, GetJobRequest const& request) {
+  internal::OptionsSpan span(options_);
```

Comment:
These should be instantiated long before the stub layer is reached.  If that is part of the bottom-up implementation strategy, then I'm sorry to (re)raise the issue.  As long as we're remembering to remove these as we build upwards, I'm happy.